config/chroot

```diff
@@ -42,4 +42,4 @@ LB_UPDATES="true"
 
 # $LB_BACKPORTS: enable backports updates
 # (Default: false)
-LB_BACKPORTS="false"
+LB_BACKPORTS="true"
```

Comment:
Need backports for codeblocks: https://packages.debian.org/search?keywords=codeblocks